buildingmotif/api/views/model.py

```diff
@@ -196,15 +197,18 @@ def validate_model(models_id: int) -> flask.Response:
                 "message": f"Libraries with ids {nonexistent_libraries} do not exist"
             }, status.HTTP_400_BAD_REQUEST
 
+        # get shacl engine if it is provided
+        shacl_engine = body.get("shacl_engine", None)
+
     # if shape_collections is empty, model.validate will default
     # to the model's manifest
-    vaildation_context = model.validate(shape_collections)
+    vaildation_context = model.validate(shape_collections, engine=shacl_engine)
```

Comment:
Another way to do this would be to have the shacle engine stored within the BuildingMOTIF singleton. This way it's a global configuration. Instead of specifying it per call. Is there a reason why we would want to use one engine only for certain calls if we have both?

Reply:
Done!